api/handlers.go

```diff
@@ -65,7 +70,12 @@ func (api *API) unpinPOST(w http.ResponseWriter, req *http.Request, _ httprouter
 		api.WriteError(w, err, http.StatusBadRequest)
 		return
 	}
-	err = api.staticDB.MarkUnpinned(req.Context(), body.Skylink)
+	sl, err := database.SkylinkFromString(body.Skylink)
```

Comment:
> Makes me wonder if we are missing some testing because the body should be empty at this point and therefore this should fail right?

I'm not following. The request body is read into the `body` struct at the beginning of the method and it's guaranteed to be not `nil`. Am I missing something?

Reply:
nah, just didn't see the toggle arrows in dark mode and it was a bit of a coincidence :)

Reply:
seems like this can be resolved. no action needed.